src/ui/public/utils/decode_geo_hash.js

```diff
@@ -1,18 +1,23 @@
 /*
- * Decodes geohash to object containing
- * top-left and bottom-right corners of
- * rectangle and center point.
- *
  * geohash.js
  * Geohash library for Javascript
  * (c) 2008 David Troy
  * Distributed under the MIT License
+ */
+
+/*
+ * @notice
+ * This product bundles geohash.js which is available under a
+ * "MIT" license.  For details, see src/ui/public/utils/decode_geo_hash.js.
+ */
+
+/**
+ * Decodes geohash to object containing
+ * top-left and bottom-right corners of
+ * rectangle and center point.
  *
- * @method refine_interval
- * @param interval {Array} [long, lat]
- * @param cd {Number}
- * @param mask {Number}
- * @return {Object} interval
+ * @param  {[type]} geohash [description]
```

Comment:
__question:__ what are these `[type]` and `[description]`?

Reply:
Haha, defaults that I forgot to replace :)